internal/service/worker.go

```diff
@@ -95,7 +95,7 @@ func (w *Worker) Process(
 	}
 
 	storage := bytes.NewBuffer([]byte{})
-	err = lazypdf.SaveToPNG(ctx, uint16(page), uint16(width), scale, bytes.NewBuffer(payload), storage)
+	err = lazypdf.SaveToPNG(ctx, uint16(page), uint16(width), scale, 96, bytes.NewBuffer(payload), storage)
```

Comment:
why 96?

Reply:
can we move it to value and put a comment near it why 96?

Reply:
or at least make it clear in the commit message why 96

Reply:
Done.